helm/leanix-k8s-connector/templates/cronjob.yaml

```diff
@@ -25,8 +25,8 @@ spec:
             securityContext:
               readOnlyRootFilesystem: true
               runAsNonRoot: true
-              runAsUser: 65534
-              runAsGroup: 65534
+              runAsUser: {{ .Values.securityContext.userId | default 65534 }}
+              runAsGroup: {{ .Values.securityContext.groupId | default 65534 }}
```

Comment:
Can you add `.Values.securityContext.userId` and ` .Values.securityContext.groupId` to the `values.yaml`?

Othwise the changes LGTM.

Reply:
I've added it to the `values.yaml`